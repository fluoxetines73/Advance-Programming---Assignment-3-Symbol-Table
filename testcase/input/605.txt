PRINT
INSERT g_1 number
RPRINT
INSERT c number