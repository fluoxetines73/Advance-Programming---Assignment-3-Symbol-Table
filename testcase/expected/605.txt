
success
g_1//0
success